checkov/terraform/checks/data/aws/IAMCredentialsExposure.py

```diff
@@ -9,7 +9,11 @@ def __init__(self):
         super().__init__(name=name, id=id)
 
     def cloudsplaining_analysis(self, policy):
-        return policy.credentials_exposure
+        excluded_actions = {
+            "ecr:GetAuthorizationToken"
```

Comment:
I would set the action as a file constant/class member

Reply:
> I would set the action as a file constant/class member

Good call--patched.